tutor/resources/styles/components/media-preview.scss

```diff
@@ -1,6 +1,9 @@
 .media-preview {
-  max-width: 300px;
-
+  max-width: 400px;
+  width: 400px;
```

Comment:
Seems redundant to have both width and max-width?

Reply:
Apparently it's only really redundant if they are exactly the same like in this case.

Reply:
heh, they're not redundant when flex box gets involved.  With that the width is what I uses for the normal size, but it'll allow expanding it to the maxWidth if there's space.  So if you don't want it to expand you have to set maxWidth as well